### PR TITLE
hoodie pls

### PR DESCRIPTION
**Pull request by ryanchou-dev**
ty have a great day

_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-1088.mdx [hunk 1]
```diff
@@ -7,7 +7,187 @@ author: Melody Yu (Video), Tyler Xiao (Video), Maggie Liu
...
+			// add the best way to the answer for alternating columns
+			colsAlternate += Math.max(sum[0], sum[1]);
+		}
+
+		io.println(Math.max(rowsAlternate, colsAlternate));
+		io.close();
+	}
+
+	// https://usaco.guide/general/io?lang=java#io-template
+	static class Kattio extends PrintWriter {
```
Comment (SansPapyrus683) [inline solutions/usaco-1088.mdx:125]: use a codesnip

Comment (SansPapyrus683) [reply solutions/usaco-1088.mdx:126]: you can replace this entire thing with just
`//CodeSnip{Kattio}`

Comment (ryanchou-dev) [reply solutions/usaco-1088.mdx:126]: CRAP

**Comment by SansPapyrus683** [comment]
Na noni, na noni doushite sayonara wa ieta no

File: solutions/usaco-1088.mdx [hunk 2]
```diff
@@ -7,7 +7,157 @@ author: Melody Yu (Video), Tyler Xiao (Video), Maggie Liu
...
+	//CodeSnip{Kattio}
+}
+```
+
+</JavaSection>
+
+<PySection>
+
+```py
+with open('planting.in', 'r') as input_file:
```
Comment (bqi343) [inline solutions/usaco-1088.mdx:133]: I just noticed this added a completely irrelevant Python solution 🤔

**Pull request merged**